artiq/test/test_rpctool.py

```diff
@@ -0,0 +1,36 @@
+import os, sys
+import asyncio
+import unittest
+
+from artiq.protocols.pc_rpc import Server
+
+class Target:
+    def output_value(self):
+        return 4125380
+
+class TestRPCTool(unittest.TestCase):
+    async def check_value(self):
+        proc = await asyncio.create_subprocess_exec(
+                            sys.executable, "-m", "artiq.frontend.artiq_rpctool", "::1", "7777", "call", "output_value",
+                            stdout = asyncio.subprocess.PIPE)
+        (value, err) = await proc.communicate()
+        self.assertEqual(value.decode('ascii').rstrip(), '4125380')
+        await proc.wait()
+
+    async def do_test(self):
+        server = Server({"target": Target()}, None, True, True)
```

Comment:
Are the default values ``description=None, builtin_terminate=False, allow_parallel=False`` not working? If they are working then please use them.